.github/workflows/check-release.yaml

```diff
@@ -0,0 +1,18 @@
+on: push
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/setup-r@master
+      - name: Install dependencies
+        run: |
+          install.packages(c("remotes", "rcmdcheck"))
+          remotes::install_deps(dependencies = TRUE)
+        shell: Rscript {0}
```

Comment:
What exactly is the `Rscript {0}` shell?

Reply:
I think it is an interface to R that allows R to execute commands from a file, instead of the normal interactive interface? No 100% sure, haven't used it.